test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
I think we need more tests here. Here's an example where lines 1 and 3 use tabs:

```ts
–⇥/**
   * More docs
–⇥ */
```

This should fail because of spaces used for indentation on line 2, right?

If possible, this PR should address #485 as well.